python-sdk/nuscenes/utils/data_classes.py

```diff
@@ -127,6 +127,9 @@ def from_file_multisweep(cls,
             else:
                 current_sd_rec = nusc.get('sample_data', current_sd_rec['prev'])
 
+        # Check that the multisweep pointcloud has the same dtype as individual pointclouds.
+        assert all_pc.points.dtype == np.float32
```

Comment:
Include an error message so the user knows what is going on when this assertion fails?

Reply:
Fixed.